latex_access/nemeth.py

```diff
@@ -115,7 +115,7 @@ def frac(self,input,start,rting=()):
             else:
                 transnum=self.translate(numerator[0],(rting[0]+numerator[2],rting[1]+1))
                 transden=self.translate(denominator[0],(rting[0]+denominator[2],rting[1]+len(transnum)+2))
-                translation="?"+transum+"/"+transden+"#"
+                translation="?"+transnum+"/"+transden+"#"
```

Comment:
Please revert this change - PR's introducing tests should not touch code they test.